packages/ui-components/src/components/AppShell/AppShell.component.tsx

```diff
@@ -41,7 +42,7 @@ export const AppShell = ({
       {contentHeading || ""}
       {embedded ? (
         <>
-          {topNavigation && topNavigation}
+          <HeaderContainer>{topNavigation && topNavigation}</HeaderContainer>
```

Comment:
Please change `{topNavigation && topNavigation}` to just `{topNavigation}` for simplicity